cooltools/api/coverage.py

```diff
@@ -55,15 +64,17 @@ def coverage(
     chunksize=int(1e7),
     map=map,
     use_lock=False,
+    clr_weight_name=None,
     store=False,
-    store_names=["cis_raw_cov", "tot_raw_cov"],
+    store_prefix="cov",
 ):
 
     """
     Calculate the sums of cis and genome-wide contacts (aka coverage aka marginals) for
     a sparse Hi-C contact map in Cooler HDF5 format.
     Note that the sum(tot_cov) from this function is two times the number of reads
-    contributing to the cooler, as each side contributes to the coverage.
+    contributing to the cooler, as each side contributes to the coverage 
+    (this only applies if clr_weight_name=None).
```

Comment:
this paranthetical is currently confusing to me.. would
Note that for raw coverage (i.e. `clr_weight_name=None`) then sum(tot_cov) calculated by this function is two times the number of reads  contributing to the cooler, as each side contributes to the coverage.

Reply:
Changed it. But isn't it also true for cis_cov, i.e.sum (cis_cov) is twice the number of cis reads? And if so should it say?

Reply:
Counting each side individually happens regardless of clr_weight_name. It's just that when clr_weight_name=None then it is calculating on raw reads, and this two times the number of reads holds exactly. Hence the previous statement "(this only applies if clr_weight_name=None)" is worded confusingly.

p.s. I'm not seeing this suggested docstring change when I view the file

Reply:
This is so weird... It's in my cloned repository

Reply:
Also I just meant that the cis counts from raw coverage will be twice the total number of cis counts in the cooler. Not just total

Reply:
Any idea why this is not showing up in the PR? It's here: https://github.com/efriman/cooltools/blob/master/cooltools/api/coverage.py#L78

Not sure what to do